thredded.gemspec

```diff
@@ -22,9 +23,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'htmlentities'
   s.add_dependency 'kaminari'
   s.add_dependency 'nokogiri'
-  s.add_dependency 'q'
   s.add_dependency 'rails', '>= 4.0.0'
   s.add_dependency 'rb-gravatar'
+  s.add_dependency 'activejob_backport' if Thredded::RailsVersion.less_than_4_2?
```

Comment:
I have a weird feeling that either this could pose a problem or someone would consider this an issue. Just a hunch ... I'm not sure though.